server/src/models/UserPersonalization.ts

```diff
@@ -0,0 +1,43 @@
+import { QueryBuilder } from 'objection';
+
+import { Domain } from './Domain';
+import { User } from './User';
+import { withDates } from './utils';
+
+export class UserPersonalization extends withDates(Domain) {
```

Comment:
```
export class UserPersonalization extends withDates(Domain)
```
should be 
```
export class UserPersonalization extends Domain
```
since UserPersonalization does not require created column which User required.

Reply:
Done!